src/gui/gui.zig

```diff
@@ -310,6 +310,9 @@ pub fn openWindow(id: []const u8) void {
 	defer updateWindowPositions();
 	for(windowList.items) |window| {
 		if(std.mem.eql(u8, window.id, id)) {
+			if (std.mem.eql(u8, id, "notification")) {
+				window.onOpenFn();
+			}
```

Comment:
This definitely not the right place to hardcode logic for specific windows.

Furthermore this produces memory leaks (as you should see if you run the game in debug mode) because this doesn't call onClose to deinit the previous window components.

A better solution might be to introduce a new function `reopenWindow` that closes the window first before attempting to open it again.

Reply:
removed it in 23dbe95